<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1381</width>
    <height>918</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>821</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:6px solid black;
border-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="seed_food">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>550</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	;
}

QPushButton:hover {
	border:3px solid black;
	background-color:rgba(105, 150, 205, 200);
}

QPushButton:pressed {
	background-color:rgba(85, 170, 255, 200);
}</string>
   </property>
   <property name="text">
    <string>喂食</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>780</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(239, 41, 41);
}

QPushButton:hover {
	border:3px solid black;
}

QPushButton:pressed {
	background-color:rgba(85, 170, 255, 200);
}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="kunkun_name">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>30</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>坤坤成长阶段</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="kunkun_growup_value">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>230</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>等级:</string>
   </property>
  </widget>
  <widget class="QLabel" name="kunkun_level">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>180</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成长值:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tishi">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>20</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tlwg Typist</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pic/shengji0.png);
color: rgb(239, 41, 41);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>890</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>版本号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>890</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volumn_switch">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>605</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="jinhua_label">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>230</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="kunkun_stage">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>134</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1（小黄鸡）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>阶段:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jinjie">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>300</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	
	
	background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
}

QPushButton:hover {
	border:1px solid black;
	background-color:rgba(105, 150, 205, 200);
}

QPushButton:pressed {
	background-color:rgba(85, 170, 255, 200);
}</string>
   </property>
   <property name="text">
    <string>进阶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dance">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>550</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	;
}

QPushButton:hover {
	border:3px solid black;
	background-color:rgba(105, 150, 205, 200);
}

QPushButton:pressed {
	background-color:rgba(85, 170, 255, 200);
}</string>
   </property>
   <property name="text">
    <string>跳舞</string>
   </property>
  </widget>
  <widget class="QLabel" name="tiaowu_tishi">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>460</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>解锁新功能：跳舞</string>
   </property>
  </widget>
  <widget class="QLabel" name="weishi_tishi">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>460</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>点击喂食来进行饲养</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bathe">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>550</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	;
}

QPushButton:hover {
	border:3px solid black;
	background-color:rgba(105, 150, 205, 200);
}

QPushButton:pressed {
	background-color:rgba(85, 170, 255, 200);
}</string>
   </property>
   <property name="text">
    <string>洗澡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pwf">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>650</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	;
}

QPushButton:hover {
	border:3px solid black;
	background-color:rgba(105, 150, 205, 200);
}

QPushButton:pressed {
	background-color:rgba(85, 170, 255, 200);
}</string>
   </property>
   <property name="text">
    <string>和朋友
玩耍</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
